lib/date_extractors/guess_extractor.dart

```diff
@@ -43,6 +43,12 @@ final _commonDatetimePatterns = [
         r'(?<date>(20|19|18)\d{2}_(01|02|03|04|05|06|07|08|09|10|11|12)_[0-3]\d_\d{2}_\d{2}_\d{2})'),
     'YYYY_MM_DD_hh_mm_ss',
   ],
+  // example: 20190620_184109.jpg
```

Comment:
I found this format of photos when going through my old photos `20190620_184109.jpg`, it seems to be used by older cameras.